subprojects/samples-tests/src/test/kotlin/org/gradle/kotlin/dsl/samples/ProjectWithBuildSrcSampleTest.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.kotlin.dsl.samples
+
+import org.hamcrest.CoreMatchers
+import org.hamcrest.MatcherAssert
+import org.junit.Test
+
+
+class ProjectWithBuildSrcSampleTest : AbstractSampleTest("project-with-buildSrc") {
+
+    @Test
+    fun `testProfile`() {
+        val output = build("printProfile").output
+        MatcherAssert.assertThat(
```

Comment:
:x: please use static imports for `MatcherAssert` and `CoreMatchers` static members